.guix-channel

```diff
@@ -2,6 +2,13 @@
   (version 0)
   (url "https://github.com/guix-science/guix-science-nonfree.git")
   (dependencies
+   (channel
+    (name nonguix)
+    (url "https://gitlab.com/nonguix/nonguix")
```

Comment:
I'd rather avoid this dependency.  It seems unnecessary so far, so let's see!

Reply:
removed